src/main/java/net/sf/jabref/logic/integrity/IntegrityCheck.java

```diff
@@ -18,18 +21,20 @@
     public static final Checker BRACKET_CHECKER = new BracketChecker();
```

Comment:
Is there a need for these public static fields? There seems to be no value in keeping a static reference...